Formula/pokeget.rb

```diff
@@ -0,0 +1,22 @@
+# Github Link: https://github.com/talwat/pokeget
+class Pokeget < Formula
+  desc "Bash script that displays cool sprites of pokemon in your terminal"
+  homepage "https://github.com/talwat/pokeget"
+  url "https://github.com/talwat/pokeget/releases/download/1.4.3/pokeget_1.4.3-src.tar"
+  sha256 "bcd94f2557856b47be7c338eddfdcc7a832f21545312a0432bb847a18b14b486"
+  license "MIT"
+  head "https://github.com/talwat/pokeget.git", branch: "main"
+
+  depends_on "make" => :build
+  on_macos do
+    depends_on "coreutils"
+  end
+
+  def install
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    system "#{bin}/pokeget", "version"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula